docs/installation.rst

```diff
@@ -1,86 +1,90 @@
---------------------------------
-
-If you're :ref:`using a virtual environment <virtualenv-install>` for Girder (which is recommended), remember to
-activate it with:
-
-.. code-block:: bash
-
-   source girder_env/bin/activate
-
-Replace ``girder_env`` with the path to your virtual environment, as appropriate.
-
-Sources
--------
+Prerequisites
+-------------
+Before running this, you must provide:
 
-Girder can be installed either from the `Python Package Index (pypi) <https://pypi.python.org/pypi>`_
-or via a `Git <https://git-scm.com/>`_ repository.
-Installing from pypi gives you the latest distributed version. Installing from git would be
-more suitable for development or to have a specific commit, or to use the latest Girder
-features before they are released in official packages.
+* A "server": an (ideally fresh) Ubuntu 18.04 system, with:
 
-Install from PyPI
-+++++++++++++++++
+  * A ``sudo``-capable user.
+  * Inbound access from the internet on TCP port 80 and 443.
+  * Outbound access to the internet on UDP port 53. Many firewalls (e.g. the
+    `AWS EC2 default security group <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-network-security.html#default-security-group>`_)
+    do not allow this by default.
+  * A DNS entry, so its public IP address is resolvable from the internet.
 
-To install the Girder distribution from the Python package index, simply run ::
+* A "controller": a machine with
+  `Ansible installed <https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html>`_
+  and SSH access to the server.
 
-    pip install girder
+* An "assetstore" for storing uploaded files on Girder. This may be either:
 
-This will install the core Girder server as a site package in your system
-or virtual environment. At this point, you might want to check the
-:doc:`configuration <configuration>` to change your plugin and logging
-paths.  In order to use the web interface, you must also install the web client
-libraries. Girder installs a Python script that will automatically build and
-install these libraries for you. Just run the following command: ::
+  * A location on the server's filesystem (which may be mounted external storage).
+  * An AWS S3 bucket.
 
-   girder build
+* Credentials for an outbound SMTP server, ideally with STARTTLS or TLS.
 
-.. note:: Installing the web client code requires Node.js. See the :ref:`Node.js installation guide <nodejs-install>`
-          for installation instructions.
+* An email address for the administrator of the system.
 
-.. note:: If you installed Girder into your system ``site-packages``, you may
-   need to run the above commands as root.
+Install Girder With Ansible
+---------------------------
+Download Template Files
++++++++++++++++++++++++
+Download the following files to a fresh directory on the controller machine:
+* `requirements.yml <https://raw.githubusercontent.com/girder/girder/master/devops/production-template/requirements.yml>`_
```

Comment:
these four bullet points don't look to be rendered correctly in markdown preview

Reply:
Will fix.